README.md

```diff
@@ -37,3 +37,166 @@ docker run -it --rm \
+```
+
+### Server Install
+
+The "server" is the machine that will have the development boards attached to
+it and/or in some instances may act as a router to connect to other boardswarm
+instances.
+
+1. Copy the `boardswarm` binary to `/usr/local/bin/` on the server.
+
+1. Create a boardswarm user and group for the boardswarm server to run as:
+   ```
+   $ sudo addgroup --system boardswarm
+   $ sudo adduser --system --ingroup boardswarm --no-create-home --disabled-password boardswarm
+   ```
+
+1. Copy the [example systemd unit](boardswarm/share/boardswarm.service) to
+   `/etc/systemd/system/boardswarm.service`.
+
+1. Copy [example udev rules](boardswarm/share/99-boardswarm.rules) to
+   `/etc/udev/rules.d/`.
+
+1. Reload udev rules and ensure they are applied:
+   ```
+   $ sudo udevadm control --reload-rules
+   $ sudo udevadm trigger
+   ```
+
+1. Copy example [`server.conf`](boardswarm/share/server.conf) to
+   `/etc/boardswarm/server.conf` on target server machine and modify as
+   required.
+
+### Server Configuration
+
+1. A mechanism for controlling power to the devices is generally required when
+   working with boards remotely. Whilst in some instances this might be handled
+   using gpio actuators, boardswarm has support for
+   [pdudaemon](https://github.com/pdudaemon/pdudaemon). Pdudaemon provides
+   support for a wide range of programmable power supplies and its use if
+   encouraged.  If using it for power mangement install and configure it to
+   switch the development boards on and off, using whichever mechanism you are
+   using (ensure that the `server.conf` `pdu` provider is configured to match the
+   configuration set in the pdudaemon configuration):
+   ```
+   $ sudo apt install pdudaemon
+   ```
+
+1. We are going to use Static JWKS based authentication, follow the process to
+   generate an EdDSA key as documented in the
+   [server README](boardswarm/README.md#static-jwks-based-authentication).
+
+1. Copy generated `auth.jwks` to `/etc/boardswarm/`
+   ```
+   $ sudo cp auth.jwks /etc/boardswarm/
+   ```
+
+### Configuring Devices
+
+For each development board you will need a section under `devices`. There is an
+example one provided in the example server configuration which can be used as a
+template. More details about this configuration in the [server README Devices
+section](boardswarm/README.md#devices).
+
+### Run the Server
+
+Reload systemd configuration, start the boardswarm server and configure it to
+run at boot:
+```
+$ sudo systemctl daemon-reload
+$ sudo systectl enable boardswarm.service
+$ sudo systemctl start boardswarm.service
+```
+
+### Client Installation
+
+1. Copy the `boardswarm-cli` binary to `/usr/local/bin/` on the client
+   development machine.
+
+1. Create a user token (expiry 01-01-2025) using the previously generated
+   `private.jwks` and then copy it to the client machine:
+   ```
+   $ rnbyc  -s '{"exp": 1735686000 }' -K private.jwks > token.jwt
```

Comment:
Just FYI, this had already expired. Perhaps the command could be updated to expire e.g a year from when its ran?